data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/repository/JpaPersistenceLayerTest.java

```diff
@@ -83,6 +72,7 @@
             new Rules<>()
     );
 
+    /*
     @Test
     void testSpringDiscoversJpaPersistenceLayer() {
```

Comment:
Should be there.  I broke it with changes I had made and dropped it on the later pile, neglecting to tag it and of course forgot about it.  I have uncommented the tests and made the one change to the configuration so it will work now.